packages/cli/src/services/__tests__/checkly-config-loader.spec.ts

```diff
@@ -45,16 +46,38 @@ describe('loadChecklyConfig()', () => {
       },
     })
   })
-  it('config TS file should export an object', async () => {
-    const configFile = './fixtures/configs/good-config.js'
+  it('config JS file should export an object', async () => {
+    const filename = 'good-config.js'
+    const configFile = `./fixtures/configs/${filename}`
     const { configDirectory, configFilenames } = splitConfigFilePath(configFile)
 
-    expect(configFilenames).toEqual(['good-config.js'])
+    expect(configFilenames).toEqual([filename])
+    expect(configDirectory).toEqual(pathToPosix(path.dirname(path.join(process.cwd(), configFile))))
+
+    const {
+      config,
+    } = await loadChecklyConfig(path.join(__dirname, 'fixtures', 'configs'), [filename])
+
+    expect(config).toMatchObject({
+      checks: {
+        checkMatch: '**/*.check.ts',
+        browserChecks: {
+          testMatch: '**/__checks__/*.spec.ts',
+        },
+      },
+    })
+  })
+  it('config from absolute path', async () => {
+    const filename = 'good-config.ts'
+    const configFile = `./fixtures/configs/${filename}`
+    const { configDirectory, configFilenames } = splitConfigFilePath(path.join(process.cwd(), configFile))
+
+    expect(configFilenames).toEqual([filename])
     expect(configDirectory).toEqual(pathToPosix(path.dirname(path.join(process.cwd(), configFile))))
 
     const {
       config,
-    } = await loadChecklyConfig(path.join(__dirname, 'fixtures', 'configs'), ['good-config.js'])
+    } = await loadChecklyConfig(path.join(__dirname, 'fixtures', 'configs'), [filename])
```

Comment:
Why don't we pass the output of `splitConfigFilePath` to the `loadChecklyConfig` call? If the goal of the test is to make sure that `splitConfigFilePath` and `loadChecklyConfig` work nicely together, then that would make more sense to me.